### ISSUE TITLE
Data of Release 

### ISSUE THREAD

**Issue by Abm-MODDER** [ZaneYork/SMAPI-Android-Installer#359]
Hi there , excuse me for not patience but anyone know when smapi will be ready, of course you know it now at alpha stage ,if anyone know when it may be ready , please tell me.

**Comment by Handyfff** [comment]
> i think only Zane know the answer...and he has all the rights to do this at his own pace... but at this point i'm sure Stardew will get the 1.6 update before we even get SMAPI for mobile 1.5...so let's just face it...we on mobile can't play with mod, just like those on console... mod is exclusive for PC players only... sad but true... ToT

I disagree, SMAPI is not fully out yet but it supports some mods. It just requires some essential dependency mods to work and then it will be out. Also don't make assumptions about 1.6 being released early, remember we waited 2+ years for 1.5. Don't worry SMAPI will be fully out in a few days hopefully.

**Comment by mymelisa** [comment]
> > i think only Zane know the answer...and he has all the rights to do this at his own pace... but at this point i'm sure Stardew will get the 1.6 update before we even get SMAPI for mobile 1.5...so let's just face it...we on mobile can't play with mod, just like those on console... mod is exclusive for PC players only... sad but true... ToT
> 
> I disagree, SMAPI is not fully out yet but it supports some mods. It just requires some essential dependency mods to work and then it will be out. Also don't make assumptions about 1.6 being released early, remember we waited 2+ years for 1.5. Don't worry SMAPI will be fully out in a few days hopefully.

i give up in waiting and get myself to play on laptop few days ago...lol...
